net-ftp/lftp/lftp-9999.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-inherit autotools eutils git-r3 libtool
+inherit autotools eutils git-r3 gnome2-utils libtool
```

Comment:
these days this is done by `xdg-utils` instead of `gnome2-utils`

Reply:
Thanks, Done.